.github/workflows/release-rollout.yml

```diff
@@ -85,3 +85,47 @@ jobs:
         SLACK_FOOTER: ""
         MSG_MINIMAL: true
 
+  composable-client-release:
+    concurrency:
+      group: composable-${{ github.ref }}
+      cancel-in-progress: true
+    strategy: 
+      fail-fast: true
+    runs-on: 
+        - self-hosted
+        - linux
+        - X64
+        - ansible-runner
+    needs: [picasso-client-release]
+    steps:
+    - name: Clean up
+      continue-on-error: true
+      run: |
+        sudo chown -R $USER:$USER $GITHUB_WORKSPACE && sudo chmod -R 777 /home/runner/_work/_tool/node/16.14.0
+        docker system prune --force --all --volumes
+
+    - uses: actions/checkout@v2
+      with: 
+        fetch-depth: 0
+
+    - name: Set env
+      run: |
+        echo "RELEASE_VERSION=$(git tag --sort=committerdate | grep -E '^v[0-9]' | tail -1 )" >> $GITHUB_ENV
+
+    - name: Run ansible playbook
+      working-directory: "./.maintain/playbooks"
+      run: |
+        ansible-playbook -l _environment_composable /opt/ansible/inventory/client-upgrade.yml -i /opt/ansible/inventory/composable-gcp.yaml  --user root -e 'ansible_python_interpreter=/usr/bin/python3'
```

Comment:
Should there be a directive runs-on : ansible-runner ? Can it be set exclusively for ansible playbook step ?

Reply:
Yes, I have already added it on L98